graphene_federation/extend.py

```diff
@@ -29,8 +29,8 @@ def decorator(Type):
             Type, "_keys"
         ), "Can't extend type which is already extended or has @key"
         # Check the provided fields actually exist on the Type.
-        assert (
-            fields in Type._meta.fields
+        assert set(fields.replace(" ", "").split(",")).issubset(
+            set(Type._meta.fields)
```

Comment:
should we do in entity.py as well? (the file just above this)

Reply:
> > > @arun-sureshkumar shall we also add support for @key(resolvable=True/False)? :)
> > 
> > 
> > We can add this!! I will add this support soon and get back to you @patrick91 !!
> 
> @arun-sureshkumar let me know if you need any help with this 😊

@patrick91 - Pls review the new changes